pom.xml

```diff
@@ -294,7 +294,7 @@
 
   <db.test.driver>com.mysql.jdbc.Driver</db.test.driver>
     <!-- For MySQL < 5.5 change 'default_storage_engine' to 'storage_engine' -->
-  <db.test.url>jdbc:mysql://127.0.0.1:3306/cgds_test?sessionVariables=default_storage_engine=InnoDB</db.test.url>
+  <db.test.url>jdbc:mysql://127.0.0.1:3306/cgds_test</db.test.url>
```

Comment:
can you switch back to `localhost`? Does the session variable need to be removed? I think I recall seeing it in a config or xml file?

Reply:
127.0.0.1 is localhost. **You cannot use localhost to connect to a mysql instance running from a docker container**. 
For new version mysql jdbc driver, default storage engine is InnoDB,no need to specify using a query string.